docs/usage/wasm-constraints.md

```diff
@@ -22,7 +22,7 @@ Instead, it is better to load individual modules as needed using
 - ssl
 - lzma
 - sqlite3
-- \_hashlib
+- \_hashlib (you need to load it via `micropip.install('hashlib')`)
```

Comment:
IMO that's a private module so it doesn't make much sense to mention it since it's not documented what it does. Instead  we should rather indicate how this affects the `hashlib` public module.

Reply:
I removed the detail of the private module and added some information about `hashlib`.